### ISSUE TITLE
Issue with checkItemRadius

### ISSUE THREAD

**Issue by raylee4204** [ChaosLeong/PinView#18]
```
<com.chaos.view.PinView
            android:id="@+id/pin_view"
            style="@style/PinWidget.PinView"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginTop="48dp"
            android:cursorVisible="false"
            android:focusable="true"
            android:focusableInTouchMode="true"
            android:imeOptions="flagNoFullscreen"
            android:inputType="numberPassword"
            android:textColor="@color/grey"
            android:textSize="18sp"
            app:itemCount="4"
            app:itemHeight="36dp"
            app:itemRadius="18dp"
            app:itemSpacing="16dp"
            app:itemWidth="36dp"
            app:layout_constraintEnd_toEndOf="parent"
            app:layout_constraintStart_toStartOf="parent"
            app:layout_constraintTop_toBottomOf="@+id/txt_description"
            app:lineColor="@color/grey"
            app:lineWidth="2dp"
            app:viewType="rectangle" />
```

I'm using the above code and the app crashes with the following:

Caused by java.lang.RuntimeException The itemRadius can not be greater than itemWidth
com.chaos.view.PinView.checkItemRadius (PinView.java:193)
com.chaos.view.PinView. (PinView.java:150)
com.chaos.view.PinView. (PinView.java:104)

The radius is exactly half the item width but the app crashes.

Happening on Galaxy S3, S7 and Pixel 2 XL

**Comment by ChaosLeong** [comment]
What version of Android on your device? @raylee4204

**Comment by raylee4204** [comment]
From Nougat to Oreo

**Comment by raylee4204** [comment]
<img width="416" alt="screen shot 2018-04-12 at 1 05 22 pm" src="https://user-images.githubusercontent.com/2085389/38692594-37c3c556-3e52-11e8-8aea-ef269345f235.png">
<img width="394" alt="screen shot 2018-04-12 at 1 05 29 pm" src="https://user-images.githubusercontent.com/2085389/38692596-39850756-3e52-11e8-975c-58c409464c74.png">
<img width="403" alt="screen shot 2018-04-12 at 1 05 37 pm" src="https://user-images.githubusercontent.com/2085389/38692598-3adc7544-3e52-11e8-9fad-65abc201af38.png">

Pulled from crashlytics

**Comment by ChaosLeong** [comment]
Can you help me test the latest changes? Because I can't reproduce this issue on Pixel 2 XL.

If you can help, please follow the steps below to get the latest changes.

__Step 1__, Add it in your root build.gradle at the end of repositories:
``` Grovvy
allprojects {
	repositories {
		...
		maven { url 'https://jitpack.io' }
	}
}
```

__Step 2__, Add the dependency
``` Grovvy
dependencies {
	implementation 'com.github.ChaosLeong:PinView:bugfix~issue_with_checkItemRadius-SNAPSHOT'
}
```

**Comment by raylee4204** [comment]
I wasn't able to repro this either

**Comment by ChaosLeong** [comment]
@raylee4204 The issue has been fixed in version 1.3.2
